README.md

```diff
@@ -42,7 +42,9 @@ vim package manager.
 NOTE: To make sure that sql.nvim work as expected, it is sometimes required
 that you set `g:sql_clib_path` or `vim.g.sql_clib_path` to where
 `libsqlite3.so` is located, and if that the case, pr or issue are welcomed to
-add to sql.nvim lookup paths for `libsqlite3.so`.
+add to sql.nvim lookup paths for `libsqlite3.so`. If you're on Windows, you'll
```

Comment:
Users might wanna know how to install sqlite3 on windows platform, like what command to run.

Reply:
I installed it by downloading a precompiled build for windows from the official website. I assume it's also possible to build it from source since they mention the source code has a Makefile